lib/spaceship/tunes/tunes_client.rb

```diff
@@ -314,10 +314,10 @@ def app_version(app_id, is_live)
 
       # We only support platforms that exist ATM
       platform = platforms.find do |p|
-        ['ios', 'osx'].include? p['platformString']
+        ['ios', 'osx', 'appletvos'].include? p['platformString']
```

Comment:
I think this would break the following use case with `deliver` right now: an app has both iOS and tvOS metadata, this way it will find the tv app and download this metadata instead of the iOS one.